<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="1"
            tests="1">

  <testsuite name="podTest"
    errors="1" failures="1" hostname="" id="" package="podTest" skipped="0"
    tests="1" time="13.83" timestamp="">
  
    <testcase name="POD Test Case" classname="podTest" time="13.83" assertions="5">

    

    

    

      <failure message="Timed out while waiting for element &lt;.po-display&gt; to be present for 5000 milliseconds. - expected &#34;visible&#34; but got: &#34;not found&#34; (5209ms)">    at Object.POD Test Case (C:\Project-INC-1\backend\tests\podTest.js:19:8)</failure>

    
    <error message="Timed out while waiting for element &lt;.po-display&gt; to be present for 5000 milliseconds. - expected [0;32m&#34;visible&#34;[0m but got: [0;31m&#34;not found&#34;[0m [0;90m(5209ms)[0m" type="error"><![CDATA[
    Error
    at Object.POD Test Case (C:\Project-INC-1\backend\tests\podTest.js:19:8)
    at Context.call (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\context.js:430:35)
    at TestCase.run (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\testcase.js:58:31)
    at Runnable.__runFn (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\index.js:693:80)
    at Runnable.run (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\runnable.js:126:21)
    at TestSuite.createRunnable (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\index.js:800:33)
    at TestSuite.handleRunnable (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\index.js:805:33)
    at C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\testsuite\index.js:693:21
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async DefaultRunner.runTestSuite (C:\Users\yeoxh\AppData\Roaming\npm\node_modules\nightwatch\lib\runner\test-runners\default.js:73:7)
    ]]></error>
    
    </testcase>
  

  
    <system-err>
       → TimeoutError

    An error occurred while running .click() command on &lt;.row.level.px-1&gt;: element not interactable
  (Session info: chrome=104.0.5112.102)
    {&#34;error&#34;:{&#34;name&#34;:&#34;ElementNotInteractableError&#34;,&#34;remoteStacktrace&#34;:&#34;&#34;},&#34;status&#34;:-1,&#34;value&#34;:null}

    Error location:
C:\Project-INC-1\backend\tests\podTest.js:
–––––––––––––––––––––––––––––––––––––––––––
 13 |       .assert.urlContains(&#39;equipmentUtilisationSnapshot&#39;, &#39;Successful login!&#39;)
 14 |       .waitForElementVisible(&#39;.EUS&#39;)
 15 |       .click({ 
 16 |         selector: &#39;.row.level.px-1&#39;,
 17 |         index: 4
–––––––––––––––––––––––––––––––––––––––––––

 → NoSuchElementError

    An element could not be located on the page using the given search parameters.

    Try fixing by :
1. Please inspect the html before the step
2. Verify if an element with the mentioned selector is present in the DOM tree

    Error location:
C:\Project-INC-1\backend\tests\userTest.js:
––––––––––––––––––––––––––––––––––––––––––––
 42 | 
 43 |     browser.refresh;
 44 |     browser.click(&#39;#Night&#39;); 
 45 |     browser.acceptAlert();
 46 |     browser.assert.textContains(
––––––––––––––––––––––––––––––––––––––––––––

    </system-err>
  

    

  
  </testsuite>
</testsuites>
